dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/StringToLongParamConverter.java

```diff
@@ -0,0 +1,27 @@
+package edu.harvard.iq.dataverse;
+
+import io.vavr.control.Try;
+import org.apache.commons.lang.StringUtils;
+
+import javax.faces.component.UIComponent;
+import javax.faces.context.FacesContext;
+import javax.faces.convert.Converter;
+import javax.faces.convert.FacesConverter;
+
+@FacesConverter(value = "strToLongConverter")
+public class StringToLongParamConverter implements Converter {
+    private Long longVal = null;
```

Comment:
It should be made local – I'm not sure what is the lifecycle of converters: whether they're singletons or not. However it's still better not to make this object stateful if not needed.